styles.css

```diff
@@ -1,31 +1,34 @@
 @import "css/benefits.css";
+
```

Comment:
we don't need empty lines between imports.
empty line should be after all imports before file content
remove please